src/mgr/MgrCommands.h

```diff
@@ -108,14 +108,14 @@ COMMAND("osd test-reweight-by-pg " \
 
 COMMAND("osd destroy "	    \
         "name=id,type=CephOsdName " \
-        "name=sure,type=CephString,req=False",
+	"name=sure,type=CephChoices,strings=--force|--yes-i-really-mean-it,req=false", \
```

Comment:
Does this mean you can do
```
ceph osd destroy osd.123 --sure=--force
```
?  It seems like what we maybe want instead is a new option,
```
name=force,type=CephBool,default=False
```
Then we'd update the code to look for the new field, and leave the optional positional one there for 2+ releases for backward compatibility?

Reply:
I agree with wanting to introduce a CephBool type, but I think it can come later?  The CephChoices style is consistent with other commands that already used that for the confirmation flag.

I'm kind of hoping that nobody would actually use the weird `--sure=--force` style, and/or the cleaned up CephBool thing would happen before releasing Nautilus

Reply:
I agree with wanting to introduce a CephBool type, but I think it can come later?  The CephChoices style is consistent with other commands that already used that for the confirmation flag.

I'm kind of hoping that nobody would actually use the weird `--sure=--force` style, and/or the cleaned up CephBool thing would happen before releasing Nautilus

Reply:
Sounds good